torch_geometric/nn/models/tgn.py

```diff
@@ -194,8 +195,28 @@ def forward(self, z_src: Tensor, z_dst: Tensor, raw_msg: Tensor,
 
 class LastAggregator(torch.nn.Module):
     def forward(self, msg: Tensor, index: Tensor, t: Tensor, dim_size: int):
-        from torch_scatter import scatter_max
-        _, argmax = scatter_max(t, index, dim=0, dim_size=dim_size)
+        if WITH_TORCH_SCATTER:
```

Comment:
Hey @puririshi98. Can we out-source this function and add a corresponding test to it? You may find the `@disableExtensions` decorator useful for this.

Reply:
yes